src/Core/Prefab/Globe/BuilderEllipsoidTile.js

```diff
@@ -33,24 +38,30 @@ BuilderEllipsoidTile.prototype.Prepare = function Prepare(params) {
 
     params.deltaUV1 = (st1 - start) * params.nbRow;
 
+    // transformation to align tile's normal to z axis
+    params.quatNormalToZ = new THREE.Quaternion().setFromAxisAngle(axisY, -(Math.PI * 0.5 - params.extent.center().latitude()));
+
     // let's avoid building too much temp objects
     params.projected = { longitudeRad: 0, latitudeRad: 0 };
 };
 
 // get center tile in cartesian 3D
-BuilderEllipsoidTile.prototype.Center = function Center(params) {
-    params.center = params.extent.center(this.tmp.coords[0])
+BuilderEllipsoidTile.prototype.Center = function Center(extent) {
+    this.tmp.coords[0]._normal = undefined;
```

Comment:
Can you please move this to `Extent` (probably in Extent.center() method) or in `Coordinates` (in the Coordinates.set())?